backend-shared/config/db/injection_test.go

```diff
@@ -0,0 +1,262 @@
+	{
+		managedEnvironment := ManagedEnvironment{
+			Managedenvironment_id: "test-managed-env-1",
+			Clustercredentials_id: clusterCredentials.Clustercredentials_cred_id,
+			Name:                  "test'env",
+		}
+		err = dbq.CreateManagedEnvironment(ctx, &managedEnvironment)
+		if !assert.NoError(t, err) {
+			return
+		}
+		retrieveManagedEnv := &ManagedEnvironment{
+			Managedenvironment_id: "test-managed-env-1",
+		}
+		err = dbq.UncheckedGetManagedEnvironmentById(ctx, retrieveManagedEnv)
+		if !assert.NoError(t, err) {
+			return
+		}
+		assert.Equal(t, managedEnvironment.Name, retrieveManagedEnv.Name)
+	}
+	assert.NoError(t, err)
+}
+
+func TestClusterUserWrongInput(t *testing.T) {
+	dbq, err := NewUnsafePostgresDBQueries(true, true)
+	if !assert.NoError(t, err) {
+		return
+	}
+	defer dbq.CloseDatabase()
+
+	ctx := context.Background()
+
+	user := &ClusterUser{
+		Clusteruser_id: "test-user-id",
+		User_name:      "samyak'scluster",
+	}
+	err = dbq.CreateClusterUser(ctx, user)
+	if !assert.NoError(t, err) {
+		return
+	}
+
+	retrieveUser := &ClusterUser{
+		User_name: "samyak'scluster",
+	}
+	err = dbq.GetClusterUserByUsername(ctx, retrieveUser)
+	if !assert.NoError(t, err) {
+		return
+	}
+	assert.NoError(t, err)
+}
+
+func TestApplicationWrongInput(t *testing.T) {
+	dbq, err := NewUnsafePostgresDBQueries(true, true)
+	if !assert.NoError(t, err) {
+		return
+	}
+	defer dbq.CloseDatabase()
+
+	ctx := context.Background()
+
+	var testClusterUser = &ClusterUser{
+		Clusteruser_id: "test-user",
+		User_name:      "test-user",
+	}
+
+	clusterCredentials := ClusterCredentials{
+		Clustercredentials_cred_id:  "test-cluster-creds-test-5",
+		Host:                        "host",
+		Kube_config:                 "kube-config",
+		Kube_config_context:         "kube-config-context",
+		Serviceaccount_bearer_token: "serviceaccount_bearer_token",
+		Serviceaccount_ns:           "Serviceaccount_ns",
+	}
+
+	managedEnvironment := ManagedEnvironment{
+		Managedenvironment_id: "test-managed-env-5",
+		Clustercredentials_id: clusterCredentials.Clustercredentials_cred_id,
+		Name:                  "my env",
+	}
+
+	gitopsEngineCluster := GitopsEngineCluster{
+		Gitopsenginecluster_id: "test-fake-cluster-5",
+		Clustercredentials_id:  clusterCredentials.Clustercredentials_cred_id,
+	}
+
+	gitopsEngineInstance := GitopsEngineInstance{
+		Gitopsengineinstance_id: "test-fake-engine-instance-id",
+		Namespace_name:          "test-fake-namespace",
+		Namespace_uid:           "test-fake-namespace-5",
+		EngineCluster_id:        gitopsEngineCluster.Gitopsenginecluster_id,
+	}
+
+	clusterAccess := ClusterAccess{
+		Clusteraccess_user_id:                   testClusterUser.Clusteruser_id,
+		Clusteraccess_managed_environment_id:    managedEnvironment.Managedenvironment_id,
+		Clusteraccess_gitops_engine_instance_id: gitopsEngineInstance.Gitopsengineinstance_id,
+	}
+
+	if err = dbq.CreateClusterCredentials(ctx, &clusterCredentials); err != nil {
```

Comment:
These should all use this:
```
	if !assert.NoError(t, err) {
			return
		}
```
or at least `assert.NoError(t, err)`), even if this is already tested elsewhere.